packages/jest-runner/schema/jest-runner-options.json

```diff
@@ -26,11 +26,6 @@
           "description": "Whether to run jest with the `--findRelatedTests` flag. When `true`, Jest will only run tests related to the mutated file per test. (See [_--findRelatedTests_](https://jestjs.io/docs/en/cli.html#findrelatedtests-spaceseparatedlistofsourcefiles)",
           "type": "boolean",
           "default": true
-        },
-        "enableBail": {
```

Comment:
This is technically a breaking change. I think we should migrate the option and log a warning here:

https://github.com/stryker-mutator/stryker-js/blob/518ebe6b946fc35138b636a015b569fe9a272ed0/packages/core/src/config/options-validator.ts#L35

Reply:
You're right. I'll do that